yarn.lock

```diff
@@ -5382,7 +5382,28 @@ polymer-analyzer@^2.0.0, polymer-analyzer@^2.0.1:
     strip-indent "^2.0.0"
     typescript "^2.2.0"
 
-polymer-build@^1.1.0, polymer-build@^1.3.0:
+polymer-build@1.4.0:
+  version "1.4.0"
+  resolved "https://registry.yarnpkg.com/polymer-build/-/polymer-build-1.4.0.tgz#840b879595c1c53ae9f33c3178d90d11608b34be"
+  dependencies:
+    "@types/mz" "0.0.31"
+    "@types/node" "^6.0.73"
+    "@types/parse5" "^2.2.32"
+    "@types/vinyl" "^2.0.0"
+    "@types/vinyl-fs" "0.0.28"
+    dom5 "^2.2.0"
+    multipipe "^1.0.2"
+    mz "^2.6.0"
+    parse5 "^2.2.2"
+    plylog "^0.5.0"
+    polymer-analyzer "^2.0.1"
+    polymer-bundler "2.0.0-pre.18"
+    polymer-project-config "^3.0.0"
+    sw-precache "^5.1.1"
+    vinyl "^1.1.1"
+    vinyl-fs "^2.4.3"
+
+polymer-build@^1.1.0:
```

Comment:
Huh, weird. Updated. Can confirm that the updated yarn.lock only results in one version of polymer-build ending up in node_modules